website/common/script/ops/pinnedGearUtils.js

```diff
@@ -95,7 +95,10 @@ export function removePinnedGearAddPossibleNewOnes (user, itemPath, newItemKey)
   // remove the old pinned gear items and add the new gear back
   removePinnedGearByClass(user);
 
-  user.items.gear.owned[newItemKey] = true;
+  const oldOwnedUserItems = { ...user.items.gear.owned };
```

Comment:
Is this one necessary?

I think, but I'm not sure that it would work with one statement instead of two, the important is that the object is completely reassigned:

```

  const newOwnedUserItems = { ...user.items.gear.owned, [newItemKey]: true };
  user.items.gear.owned = newOwnedUserItems; 
```

Reply:
yeah, it will work in that way to, just for better and comprehensive code I create another not necessary (maybe just for now) variable. I will remove it.